crates/router/src/connector/adyenplatform.rs

```diff
@@ -401,6 +399,25 @@ impl api::IncomingWebhook for Adyenplatform {
         }
     }
 
+    fn get_webhook_api_response(
+        &self,
+        _request: &api::IncomingWebhookRequestDetails<'_>,
+        error_kind: Option<IncomingWebhookFlowError>,
+    ) -> CustomResult<services::api::ApplicationResponse<serde_json::Value>, errors::ConnectorError>
+    {
+        if error_kind.is_some() {
+            Ok(services::api::ApplicationResponse::JsonWithHeaders((
+                serde_json::Value::Null,
+                vec![(
+                    "x-http-code".to_string(),
+                    Maskable::Masked(Secret::new("404".to_string())),
+                )],
```

Comment:
Do we still need this custom header? If yes, then where would be monitoring this (Grafana dashboard, etc.)? Or can we have metrics instead?